packages/react/src/components/ThemeProvider/index.tsx

```diff
@@ -1,23 +1,42 @@
 import * as React from 'react';
+import { DirectionProvider } from '@radix-ui/react-direction';
 
 import { createTheme, Theme } from '@aws-amplify/ui';
 
 import { AmplifyContext } from './AmplifyContext';
 
 export type ColorMode = 'system' | 'light' | 'dark';
+export type Direction = 'ltr' | 'rtl';
```

Comment:
I think [auto](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/dir) is a valid option, does this conflict with Radix's options?

Reply:
Yeah, unfortunately radix only allows 'ltr' and 'rtl'
https://github.com/radix-ui/primitives/blob/main/packages/react/direction/src/Direction.tsx#L3